test/integration/scripting_spec.js

```diff
@@ -30,9 +30,12 @@ describe("Interaction", () => {
     it("must check that first text field has focus", async () => {
       await Promise.all(
         pages.map(async ([browserName, page]) => {
+          await page.waitForFunction(
+            "window.PDFViewerApplication.opendocfinished === true"
```

Comment:
I think we don't need the `=== true` check here since if it's not defined `window.PDFViewerApplication.opendocfinished` should be undefined (falsy).